src/Resources/config/messenger.xml

```diff
@@ -12,5 +12,11 @@
 
             <tag name="fos_elastica.pager_persister" persisterName="async" />
         </service>
+
+        <service id="fos_elastica.async_parsist_page_handler" class="FOS\ElasticaBundle\Message\Handler\AsyncPersistPageHandler">
+            <argument type="service" id="fos_elastica.async_pager_persister" />
+
+            <tag name="messenger.message_handler" bus="command.bus" />
```

Comment:
The `bus` is configurable via config. Shouldnt the tag be added via a compiler pass?
```yaml
fos_elastica:
  messenger:
    message_bus: query.bus
```